website/static/js/share/stats.js

```diff
@@ -121,6 +124,7 @@ Stats.controller = function(vm) {
             url: '/api/v1/share/stats/?' + $.param({q: self.vm.query()}),
             background: true
         }).then(function(data) {
+            data.charts.shareDonutGraph.onclick = function (d, element) { return getSourceName(d, element, self.vm) };
```

Comment:
Simple patch [here](https://github.com/erinspace/osf.io/blob/feature/share/search_updates/website/static/js/share/stats.js#L115)

```javascript
self.graphs[divId] = graphFunction(self.vm.statsData, self.vm);
```
The `vm.statsData` could be omitted in that case but it may be a bit cleaner to explicitly pass it.